gravitee/values.yaml

```diff
@@ -25,6 +25,19 @@ oauth:
 inMemoryAuth:
   enabled: true
 
+# Default password "admin", use bcrypt to generate a new one
+adminPasswordBcrypt: aaa
```

Comment:
I think it would be useful to keep the default value from `gravitee.yml` here.

Reply:
Sorry, leftover from testing. Fixed.